db/migrate/20160308093916_migrate_security_protocol_atribute_to_endpoints.rb

```diff
@@ -0,0 +1,40 @@
+class MigrateSecurityProtocolAtributeToEndpoints < ActiveRecord::Migration[5.0]
+  class Endpoint < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Migrating Security Protocol attribute to Endpoints") do
+      ExtManagementSystem.all.each do |ems|
+        next if ems.security_protocol.nil?
+        endpoint = Endpoint.where(
+          :resource_type => "ExtManagementSystem",
+          :resource_id   => ems.id).first_or_create
```

Comment:
Isn't that what `:role => "default"` is doing. Finding the default role for an `Endpoint`